release-please-config.json

```diff
@@ -0,0 +1,33 @@
+{
+  "bootstrap-sha": "0ef1fc28d366b74d98b73b5d6334ffdc75342fe2",
```

Comment:
Where comes this SHA from?
if it is intended that someone else the you releases a readme/doc would be nice.

Reply:
This is the sha for the last manual release. It is used by release-please to know when the automated history starts and the manual release history ends.